settings.py

```diff
@@ -38,12 +36,13 @@
 DEV_MODE = os.environ['SERVER_SOFTWARE'].startswith('Development')
 UNIT_TEST_MODE = os.environ['SERVER_SOFTWARE'].startswith('test')
 
+#setting GOOGLE_CLOUD_PROJECT manually in dev mode
+APP_ID = os.environ.get('GOOGLE_CLOUD_PROJECT', 'dev') if DEV_MODE else os.environ['GOOGLE_CLOUD_PROJECT']
 
-APP_ID = app_identity.get_application_id()
 SITE_URL = 'http://%s.appspot.com/' % APP_ID
 CLOUD_TASKS_REGION = 'us-central1'
 
-if APP_ID == 'testbed-test':
+if UNIT_TEST_MODE == 'testbed-test':
```

Comment:
`UNIT_TEST_MODE` will be either `TRUE` or `FALSE`, depending on, if the value of  `os.environ['SERVER_SOFTWARE']` starts with `test` (See line 39). Hence, I would recommed that you change this line to following:-
`if UNIT_TEST_MODE:`